Eatery Blue/UI/EateryCard/EaterySmallCardView.swift

```diff
@@ -74,11 +114,75 @@ class EaterySmallCardView: UIView {
 
             NotificationCenter.default.post(
                 name: NSNotification.Name("favoriteEatery"),
-                object: nil
+                object: nil,
+                userInfo: ["favorited": metadata.isFavorite]
             )
         }
     }
 
+    private func setUpStackView() {
+        stackView.axis = .vertical
+        stackView.spacing = 4
+    }
+
+    private func configureImageView() {
+        guard let eatery else { return }
+
+        imageView.kf.setImage(with: eatery.imageUrl)
+        imageView.hero.id = eatery.imageUrl?.absoluteString
+        imageView.alpha = eatery.isOpen ?  1 : 0.5
+    }
+
+    private func configureSubtitleLabels() {
+        guard let eatery else { return }
+
+        stackView.subviews.forEach { view in
+            view.removeFromSuperview()
+        }
+
+        let openStatusLabel = UILabel()
+        setUpSubtitleLabel(openStatusLabel)
+
+        let walkingTimeLabel = UILabel()
+        setUpSubtitleLabel(walkingTimeLabel)
+
+        LocationManager.shared.$userLocation
+            .sink { userLocation in
+                let subtitle = EateryFormatter.default.formatEatery(
+                    eatery,
+                    style: .medium,
+                    font: .preferredFont(for: .footnote, weight: .medium),
+                    userLocation: userLocation,
+                    date: Date()
+                ).first?.split(seperateBy: " · ")
+
+                openStatusLabel.attributedText = subtitle?.last
+                walkingTimeLabel.attributedText = subtitle?.first
+            }
+            .store(in: &cancellables)
```

Comment:
I think this can be done a bit better reflecting how things are typically done in combine also add a deinit to remove all cancellables and remove that observer
```suggestion
        LocationManager.shared.$userLocation
        .map { userLocation -> (NSAttributedString?, NSAttributedString?) in
            let subtitle = EateryFormatter.default.formatEatery(
                eatery,
                style: .medium,
                font: .preferredFont(for: .footnote, weight: .medium),
                userLocation: userLocation,
                date: Date()
            ).first?.split(seperateBy: " · ")
            return (subtitle?.first, subtitle?.last)
        }
        .receive(on: DispatchQueue.main)
        .sink { [weak self] first, last in
            openStatusLabel.attributedText = last
            walkingTimeLabel.attributedText = first
        }
        .store(in: &cancellables)
```